added terraform and jekinsfiles

PramodaHS opened the pull request:
Hi Sir, I have completed the Terraform setup for deploying the S3 static website using Jenkins. The changes are tested in us-1048, and I’m requesting approval to merge into main. This will enable automated infrastructure deployment, with terraform apply running only after the merge. Please review and approve if everything looks good. Let me know if any changes are needed.

Regards, 
Pramodhs

PramodaHS committed:
added terraform and jekinsfiles

HKHemanthKumar reviewed (changes requested):
Give walkthrough of your code After I will approve

PramodaHS committed:
added backend file and modified jenkins file for backend ans also added ReadMe file for my project

PramodaHS commented:
Hi sir I have added backend terraform file configured with jenkins file also and added ReadMe file for my project. The changes are tested in us-1048, and I’m requesting approval to merge into main. This will enable automated infrastructure deployment, with terraform apply running only after the merge. Please review and approve if everything looks good. Let me know if any changes are needed.
Regards 
Pramodhs

PramodaHS committed:
modified ReadMe file
